runner/dl_checkpoints.sh

```diff
@@ -24,13 +24,15 @@ function download_alpha_models() {
     printf "\nDownloading unrestricted models...\n"
 
     # Download text-to-image and image-to-image models.
-    huggingface-cli download SG161222/RealVisXL_V4.0_Lightning --include "*.fp16.safetensors" "*.json" "*.txt" --exclude ".onnx" ".onnx_data" --cache-dir models
     huggingface-cli download ByteDance/SDXL-Lightning --include "*unet.safetensors" --cache-dir models
     huggingface-cli download timbrooks/instruct-pix2pix --include "*.fp16.safetensors" "*.json" "*.txt" --cache-dir models
 
     # Download upscale models
     huggingface-cli download stabilityai/stable-diffusion-x4-upscaler --include "*.fp16.safetensors" --cache-dir models
 
+    # Download speech-to-text models.
+    huggingface-cli download openai/whisper-large-v3 --include "*.safetensors" "*.json" --cache-dir models
```

Comment:
Have we evaluated frameworks like this: https://github.com/Vaibhavs10/insanely-fast-whisper

From my limited understanding, the frameworks have various optimizations that speed up processing significantly on CPU and GPU.

Reply:
As far as I know only the base model and transformers pipeline has been looked at for this initial implementation.  Thinking that linked framework can be run as a separate "model" offering on the network similar to lightning and non-lightning models for text2image.  Will require some logic switching to load the pipeline differently but if it really is significantly faster with no or minimal trade offs on quality and/or allows GPUs with lower VRAM to run larger whisper models it would be worth looking at more!

Reply:
We have some network optimizations coming in a future release. Will keep an eye on this implementation for future improvement.